changelogs/fragments/549-2.10-removal.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
I'm not sure it's worthwhile to add a clog fragment for internal refinements that aren't visible to users (we already bail out if someone tries to build for unsupported ansible versions), but I'll leave that up to your discretion.

Reply:
I'm going to change it to a `trivial` one, so it won't show up in the human-readable changelog.

Reply:
```suggestion
trivial:
```

Reply:
TIL, I didn't know `trivial` existed!